migration/sql-files/103-add-rules-to-wit.sql

```diff
@@ -0,0 +1,98 @@
+-- Add action rules fields to work_item_types. The values are
+-- optional, so they may be null and are being kept at null
+-- on existing WITs. Values for current templates will be
+-- imported from the template on next launch.
+ALTER TABLE work_item_types ADD COLUMN trans_rule_key text;
+ALTER TABLE work_item_types ADD COLUMN trans_rule_argument text;
+
+-- migrate existing work items
+
+-- Agile Template - Impediment
+update work_items set fields = fields || '{"system.metastate": "mNew", "system.boardcolumns": ["7389fa7d-39c8-4865-8094-eda9a7836161", "7e3bbf09-44c4-419e-8d43-10e00400ca80"] }' WHERE fields @> '{"system.state": "New"}' AND type='03b9bb64-4f65-4fa7-b165-494cd4f01401';
+update work_items set fields = fields || '{"system.metastate": "mOpen", "system.boardcolumns": ["7063ae46-994d-49e8-99f9-2ad867dd340e", "29124ef0-d651-47c4-84a7-28acb7a4ab7a"] }' WHERE fields @> '{"system.state": "Open"}' AND type='03b9bb64-4f65-4fa7-b165-494cd4f01401';
+update work_items set fields = fields || '{"system.metastate": "mInprogress", "system.boardcolumns": ["f7243e68-1d2b-4256-b6e7-3c657c944ff1", "a30fc0e0-bfa9-43b1-a83d-b62ae2d5d0f7"] }' WHERE fields @> '{"system.state": "In Progress"}' AND type='03b9bb64-4f65-4fa7-b165-494cd4f01401';
+update work_items set fields = fields || '{"system.metastate": "mResolved", "system.boardcolumns": ["9f780106-4d71-41bf-b017-001ca7e19162", "ca1ea842-1650-4435-88b3-560e5bf47d42"] }' WHERE fields @> '{"system.state": "Resolved"}' AND type='03b9bb64-4f65-4fa7-b165-494cd4f01401';
+update work_items set fields = fields || '{"system.metastate": "mClosed", "system.boardcolumns": [] }' WHERE fields @> '{"system.state": "Closed"}' AND type='03b9bb64-4f65-4fa7-b165-494cd4f01401';
+-- Agile Template - Defect
+update work_items set fields = fields || '{"system.metastate": "mNew", "system.boardcolumns": ["7389fa7d-39c8-4865-8094-eda9a7836161, 7e3bbf09-44c4-419e-8d43-10e00400ca80"] }' WHERE fields @> '{"system.state": "New"}' AND type='fce0921f-ea70-4513-bb91-31d3aa8017f1';
+update work_items set fields = fields || '{"system.metastate": "mOpen", "system.boardcolumns": ["7063ae46-994d-49e8-99f9-2ad867dd340e, 29124ef0-d651-47c4-84a7-28acb7a4ab7a"] }' WHERE fields @> '{"system.state": "Open"}' AND type='fce0921f-ea70-4513-bb91-31d3aa8017f1';
```

Comment:
The quotes are incorrect.
```
"system.boardcolumns": ["7063ae46-994d-49e8-99f9-2ad867dd340e, 29124ef0-d651-47c4-84a7-28acb7a4ab7a"]
```
should be
```
"system.boardcolumns": ["7063ae46-994d-49e8-99f9-2ad867dd340e", "29124ef0-d651-47c4-84a7-28acb7a4ab7a"]
```

The API returns single value in the board column instead of multiple values due to incorrect quotes.
The same issue exists for all workitem type except Impediment.

Reply:
Good find! Fixed in 4fffaea.